### PR TITLE
[14.0][ADD] custom_breeding_report: Breeding reports.

### PR DESCRIPTION
**Pull request by Berezi**

### CODE REVIEW

**Review by oihane — CHANGES_REQUESTED** [review]
```
2023-11-23 10:27:14,015 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_mother_lineage_relation_xlsx'.
  warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
 
2023-11-23 10:27:14,027 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/lineage_percentage_report_xlsx.xml 
2023-11-23 10:27:14,030 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_lineage_percentage_xlsx'.
  warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
 
2023-11-23 10:27:14,038 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/stock_picking_batch_report_xlsx.xml 
2023-11-23 10:27:14,041 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_stock_picking_batch_xlsx'.
  warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
 
2023-11-23 10:27:14,048 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/breeding_general_summary_xlsx.xml 
2023-11-23 10:27:14,053 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_breeding_general_summary_xlsx'.
  warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)

```

**Comment by Berezi** [comment]
> ```
> 2023-11-23 10:27:14,015 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_mother_lineage_relation_xlsx'.
>   warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
>  
> 2023-11-23 10:27:14,027 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/lineage_percentage_report_xlsx.xml 
> 2023-11-23 10:27:14,030 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_lineage_percentage_xlsx'.
>   warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
>  
> 2023-11-23 10:27:14,038 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/stock_picking_batch_report_xlsx.xml 
> 2023-11-23 10:27:14,041 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_stock_picking_batch_xlsx'.
>   warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
>  
> 2023-11-23 10:27:14,048 3725011 INFO pt_20220919B_copia2610b odoo.modules.loading: loading custom_breeding_report/report/breeding_general_summary_xlsx.xml 
> 2023-11-23 10:27:14,053 3725011 WARNING pt_20220919B_copia2610b py.warnings: /opt/odoo/v14/core/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'report_breeding_general_summary_xlsx'.
>   warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
> ```

Zuzenduta. Eskerrikasko.

File: custom_breeding_report/report/breeding_general_summary_xlsx.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (oihane) [inline custom_breeding_report/report/breeding_general_summary_xlsx.xml]: Esto no es necesario

File: custom_breeding_report/report/stock_picking_batch_xlsx.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
+
+import logging
+
+from odoo import _, fields, models
+
+_logger = logging.getLogger(__name__)
```
Comment (oihane) [inline custom_breeding_report/report/stock_picking_batch_xlsx.py]: No se usa

Comment (Berezi) [reply custom_breeding_report/report/stock_picking_batch_xlsx.py]: Kenduta. Mertzi

**Review by anajuaristi — APPROVED** [review]

**Review by anajuaristi — APPROVED** [review]

**Pull request merged**